readme.md

```diff
@@ -69,7 +69,6 @@ See: [Incremental rendering](doc/incrementalrendering.md)
 ## Planned Features
 
 * Isometric view
-* Skin support
```

Comment:
If there are other skin mods or methods of support still required, I can add this back.